tests/forms_tests/tests/test_forms.py

```diff
@@ -2377,3 +2377,46 @@ def test_baseform_repr_dont_trigger_validation(self):
         self.assertRaises(AttributeError, lambda: p.cleaned_data)
         self.assertFalse(p.is_valid())
         self.assertEqual(p.cleaned_data, {'first_name': 'John', 'last_name': 'Lennon'})
+
+    def test_field_name_with_hidden_input(self):
+        """
+        BaseForm._html_output() uses inconsistent formatting for normal row
+        """
+
+        class SomeForm(Form):
+            hidden1 = CharField(label='hidden1', widget=HiddenInput, initial='h1')
+
```

Comment:
no newline between fields (see style of other forms)
Also, I don't think `label` and `initial` need to be specified. Try to include only the minimum functionality that's needed to reproduce the error and prove the regression is fixed.